app/components/album-art.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import layout from '../templates/components/album-art';
+
+export default Ember.Component.extend({
+  layout: layout,
+  classNames: ["album-art"],
+  classNameBinding: ["loading"],
```

Comment:
I'm getting strange errors too. I pulled the latest version but didn't `npm`/`bower` `install`.